spec/request/folders_request_spec.rb

```diff
@@ -65,4 +65,34 @@
       end
     end
   end
+
+  describe '#destroy' do
+    context 'with valid folder id' do
+      let(:id) { 613_557 }
+
+      it 'response to DELETE #destroy' do
+        VCR.use_cassette("sm/folders/#{user_id}/delete_valid") do
+          delete "/v0/messaging/health/folders/#{id}"
+        end
+
+        expect(response).to be_success
+        expect(response).to have_http_status(:no_content)
+      end
+    end
+
+    context 'with non-existing id' do
```

Comment:
In this particular case, it would appear so.

`GET https://staging.vets.gov/api/v0/messaging/health/folders/-1`

```json
{
  "data": {
    "id": "-1",
    "type": "folders",
    "attributes": {
      "folder_id": -1,
      "name": "Sent",
      "count": 91,
      "unread_count": 0,
      "system_folder": true
    },
    "links": {
      "self": "https:\/\/staging.vets.gov\/api\/v0\/messaging\/health\/folders\/-1"
    }
  }
}
```

`DELETE https://staging.vets.gov/api/v0/messaging/health/folders/-1`

Is not implemented on staging yet, but when I test on my local server I get "Entity Not Found"